### PR TITLE
Statistic and Proportional Bar components

### LINKED ISSUE #297: Proportional bar chart component
Convert `ProportionalBar` component to Typescript in the new `charts` directory. It should be able to use existing Tooltip and Legend components.

### LINKED ISSUE #298: Statistic component
Convert `Statistic` component to TS and add tests (currently none exist). Adapt to new styles as needed.

### PR DESCRIPTION
**Pull request by macfarlandian**
## Description of the change

Not much to see here ... these are adapted mostly as-is from v1 with some light Typescript conversion. For consistency I am now using our MeasureWidth component instead of the Semiotic `responsiveWidth` feature for the chart, but otherwise it is functionally identical. 

Not used anywhere in the UI yet but I did add some tests at least. For reference, this is what they looked like in v1:

Screen Shot 2021-01-27 at 1 45 19 PM

This unblocks the population sections at the top of every page, which is a larger chunk of work that will follow in the next PR. 

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #297, closes #298 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: spotlight-client/src/UiLibrary/typography.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+const [mobileMin] = breakpoints.mobile;
+const [desktopMin] = breakpoints.desktop;
+
+/**
+ * Implements fluid typography technique to scale text size
+ * by viewport from min to max
+ * @see <https://css-tricks.com/simplified-fluid-typography/>
+ */
+export function fluidFontSizeStyles(
```
Comment (jovergaag) [inline spotlight-client/src/UiLibrary/typography.ts:34]: Cool!

**Review by jovergaag — APPROVED** [review]
